modules/virt-delete-vm-web.adoc

```diff
@@ -1,22 +1,33 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virt-delete-vms.adoc
+// * virt/managing-vms/virt-delete-vms.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-delete-vm-web_{context}"]
 
 = Deleting a virtual machine using the web console
 
-Deleting a virtual machine permanently removes it from the cluster.
+Deleting a virtual machine (VM) permanently removes it from the cluster.
+
+If the VM is delete protected, the *Delete* action is disabled in the VM's *Actions* menu. To delete the VM you must first disable the VM's delete protection setting.
+
+
 
 .Procedure
 
-. In the {product-title} console, click *Virtualization* -> *VirtualMachines* from the side menu.
+. From the {product-title} web console, choose your view:
 
-. Click the Options menu {kebab} beside a virtual machine and select *Delete*.
+    * For a virtualization-focused view, select *Administrator* → *Virtualization* → *VirtualMachines*.
+
+    * For a general view, navigate to *Virtualization* → *VirtualMachines*.
```

Comment:
Just checking - is it actually necessary to show both ways? I could see yes if some users might see it different ways than others. But in the interest of reducing complexity, would be better if we could only show one way (see https://spaces.redhat.com/display/DOCS/Too+many+decisions#Toomanydecisions-Usershavetomakeadecisionmid-procedure)